charts/kubeadmiral/_admiral_control_plane/crds/core_kubeadmiral_io/core.kubeadmiral.io_clustercollectedstatuses.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Are these files copied from the repo? Should we have a separate script to build the helm chart in CI and copy the CRD files to this directory in the CI instead?

Reply:
There is a generated script file(`hack/generate-groups.sh`) for CRD, which has been updated here. It will be synchronized and updated to the Helm chart directory every time CRD is generated.

Reply:
If you insist to commit the generated files twice, please add the new paths to .gitattributes to mark them as generated.

Reply:
and [typos.toml](https://github.com/kubewharf/kubeadmiral/blob/02a8fc94bdc4431ce369bb80d935627c804e6cfc/hack/typos.toml#L2) as well

Reply:
The `.gitattributes` and `typos.toml` has already been updated.